avm/res/container-service/managed-cluster/agent-pool/main.bicep

```diff
@@ -46,6 +46,9 @@ param gpuInstanceProfile string?
 @description('Optional. Determines the placement of emptyDir volumes, container runtime data root, and Kubelet ephemeral storage.')
 param kubeletDiskType string?
 
+@description('Optional. Linux OS configuration.')
+param linuxOSConfig linuxOSConfigType?
```

Comment:
@AlexanderSehr already mentioned this but leaving an explicit comment here so we don't forget to update this to a resource derived type.